src/features/catalog/CatalogPage.tsx

```diff
@@ -0,0 +1,9 @@
+import ProductList from './components/ProductList/ProductList';
+
+export default function Catalog() {
+  return (
+    <>
+      <ProductList />
+    </>
+  );
+}
```

Comment:
In my branch were I made Product card, I use this:

```suggestion
import { Grid, Box, TextField } from '@mui/material';
import ProductList from './components/ProductList/ProductList';

export default function CatalogPage() {
  return (
    <Box
      component="div"
      sx={{
        maxWidth: { lg: '1108px', md: '850px', xs: '90%' },
        display: 'flex',
        flexDirection: 'column',
        gap: 2,
        margin: 'auto',
        py: { lg: 8, md: 6, xs: 4 },
      }}
    >
      <Grid container spacing={3}>
        <Grid size={3}>
          <Box sx={{ p: 1 }}>
            <TextField fullWidth label="Search shelter" variant="outlined" margin="normal" />
          </Box>
        </Grid>
        <Grid size={9}>
          <ProductList />
        </Grid>
      </Grid>
    </Box>
  );
}
```

Reply:
Thanks, fixed catalog appearance